show/reboot_cause.py

```diff
@@ -9,7 +10,8 @@
 
 
 PREVIOUS_REBOOT_CAUSE_FILE_PATH = "/host/reboot-cause/previous-reboot-cause.json"
-
+STATE_DB = 6
```

Comment:
The DP indexes are part of the swsscommon definition:
```
>>> import swsscommon.swsscommon as s
>>> s.CHASSIS_STATE_DB
13
>>> s.STATE_DB
6
```
There is no need to redefine them